File: content/api-documentation/how-to/account/examples/gainloss.js
const financefeast = require('@financefeasthq/financefeast-trade-api')
const financefeast = new financefeast()

// Get account information.
financefeast.getAccount().then((account) => {
	// Calculate the difference between current balance and balance at the last market close.
    const balanceChange = account.equity - account.last_equity

    console.log('Today\'s portfolio balance change:', balanceChange)
})

File: content/api-documentation/how-to/account/examples/gainloss.go
package main

import (
	"fmt"
	"log"

	"github.com/financefeasthq/financefeast-trade-api-go/financefeast"
)

func main() {
	financefeast.SetBaseUrl("https://paper-api.financefeast.markets")

	// Get account information.
	account, err := financefeast.GetAccount()
	if err != nil {
		log.Fatalln(err)
	}

	// Calculate the difference between current balance and balance at the last market close.
	balanceChange := account.Equity.Sub(account.LastEquity)

	fmt.Println("Today's portfolio balance change:", balanceChange)
}
